### PR TITLE
[ISSUE #220] Add HTTP Sink Connector Document

### PR DESCRIPTION
**Pull request by cnzakii**
[Doc] Add HTTP Sink Connector Document #220

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/design-document/03-connect/02-http-connector.md [hunk 1]
```diff
@@ -64,4 +64,148 @@ curl --location --request POST 'http://localhost:3755/test' \
     "datacontenttype":"text/plain",
     "data": "testdata"
 }'
-```
\ No newline at end of file
+```
+
+
+
+## HTTP Sink Connector
```
Comment (Pil0tXia) [inline i18n/zh/docusaurus-plugin-content-docs/current/design-document/03-connect/02-http-connector.md:71]: Please remove similar redundant lines.

**Review by Pil0tXia — CHANGES_REQUESTED** [review]

File: i18n/zh/docusaurus-plugin-content-docs/current/design-document/03-connect/02-http-connector.md (hunk 1 above)
Comment (cnzakii) [reply i18n/zh/docusaurus-plugin-content-docs/current/design-document/03-connect/02-http-connector.md]: Resolved at https://github.com/apache/eventmesh-site/pull/221/commits/1fbeb926ed8b2d67176ba3e147fdc5858dc61517

File: docs/design-document/03-connect/02-http-connector.md [hunk 2]
```diff
@@ -5,12 +5,15 @@
...
-  - (required) `path`, path of the API.
-  - (required) `port`, port of the API.
-  - `idleTimeout`, idle TCP connection timeout in seconds. A connection will timeout and be closed if no data is received nor sent within the `idleTimeout` seconds. The default is 0, which means don't timeout.
+- Please configure the source connector in `/resource/source-config.yml`, only the configuration under `connectorConfig`
+  is described here:
+    - `connectorName`, name of the connector.
+    - (required) `path`, path of the API.
+    - (required) `port`, port of the API.
+    - `idleTimeout`, idle TCP connection timeout in seconds. A connection will timeout and be closed if no data is
+      received nor sent within the `idleTimeout` seconds. The default is 0, which means don't timeout.
```
Comment (Pil0tXia) [inline docs/design-document/03-connect/02-http-connector.md:16]: intended line-break?

Comment (cnzakii) [reply docs/design-document/03-connect/02-http-connector.md:16]: This is the result of global formatting and doesn't seem to affect the presentation, does it need to be one line?

Comment (Pil0tXia) [reply docs/design-document/03-connect/02-http-connector.md:16]: Yes, it should be an unordered list here.

Comment (Pil0tXia) [reply docs/design-document/03-connect/02-http-connector.md:16]: No I think.

Comment (cnzakii) [reply docs/design-document/03-connect/02-http-connector.md]: Resolved at https://github.com/apache/eventmesh-site/pull/221/commits/49ec87ea9eac8afa45f931d7f102f0fbdc7c5646

**Review by Pil0tXia — APPROVED** [review]
LGTM

**Pull request merged**